hw/ip/kmac/lint/kmac.waiver

```diff
@@ -4,4 +4,17 @@
 #
 # waiver file for KMAC
 
+waive -rules {HIER_NET_NOT_READ NOT_READ INPUT_NOT_READ} -location {kmac.sv} -regexp {'reg_state_valid' in module .* is not read} \
+      -comment "state values are guarded by the FSM, so kmac_staterd does not use the valid signal"
 
+waive -rules {HIER_NET_NOT_READ NOT_READ INPUT_NOT_READ} -location {kmac.sv} \
+    -regexp {'tlram_addr' is not read} \
+    -comment "MSG_FIFO ignores the exact memory address but use strobe for alignement"
+
+waive -rules {HIER_NET_NOT_READ NOT_READ INPUT_NOT_READ} \
+    -location {kmac_msgfifo.sv} -regexp {'packer_wmask.* is not read} \
+    -comment "MSG_FIFO uses only the first bit in a byte"
+
+waive -rules {HIER_NET_NOT_READ NOT_READ INPUT_NOT_READ} \
```

Comment:
do you actually need this one? could you try taking it out? I think just by having `unused_` the tool automatically waives them.

Reply:
yeah right,  `assign unused_*` should do the job here...

Reply:
Oh.. so, I just connect the sub-module's output to this unused_* signal. So, in order to be waived, it should be explictly assigned?

Reply:
yes exactly. the lint rules detect signals that have an `unused_` prefix, and ignore those if they are not read. that way, we can get rid of the waiver and fix this directly in RTL...